common/src/main/resources/docker/ecs.bom

```diff
@@ -0,0 +1,231 @@
+brooklyn.catalog:
+  version: "2.1.0-SNAPSHOT" # CLOCKER_VERSION
+  publish:
+    description: |
+      Resources for working with Docker and Amazon ECS from Apache Brooklyn
+    license_code: APACHE-2.0
+    overview: README.md
+
+  items:
+
+  - id: ecs-cluster-template
+    name: "EC2 Container Service Docker Cluster"
+    description: |
+      Creates a cluster of Docker engines for use with Amazon ECS
+    itemType: template
+    iconUrl: https://upload.wikimedia.org/wikipedia/commons/thumb/a/af/AWS_Simple_Icons_Compute_ECS.svg/240px-AWS_Simple_Icons_Compute_ECS.svg.png
+    item:
+      services:
+        - type: ecs-cluster
+
+  - id:  ecs-cluster
```

Comment:
It's scary that we're repeating all of these brooklyn.parameters in different entities in this same file.

What is the purpose of having `ecs-cluster` and `docker-cluster` as separate items?